docs/production-deployment/cloud/billing-and-cost.mdx

```diff
@@ -76,3 +78,7 @@ The Invoices table has the following information:
 - Balance due - Balance remaining after credits are granted
 
 You have the ability to download your Invoices by clicking the download icon by the date. 
+
+## Cost details {#cost-details}
+
+As an Account Owner, you will have access to the cost column under the Usage page. This column will allow you to see cost per namespace. Note that the cost details are unavailable for the last 90 or 120 days time period filters. To view cost per namespace, you must select Current Period, Last Month, Two Months Ago, or Three Months Ago. 
```

Comment:
See previous comment and please make sure you've signed our contributor agreement.

```suggestion
Account Owners may view the cost column on the Usage page, which shows the cost-per-Namespace for their account.
Select a period such as Current Period, Last Month, Two Months ago, etc.
You cannot combine periods, such as "last 90 days" or "last 120 days", to view accumulated costs.
```